aea/contracts/base.py

```diff
@@ -191,3 +192,25 @@ def get_state(
         :return: the tx
         """
         raise NotImplementedError
+
+
+def _try_to_register_contract(configuration: ContractConfig):
```

Comment:
Looks like this needs to be moved elsewhere to avoid circular imports, e.g. `aea.contracts.helpers`

Reply:
merged #1697 in this branch. Better to merge the other PR before this one.